user/list.md

```diff
@@ -3,3 +3,34 @@ description: How to list your company's users
 ---
 
 # List users
+
+You can search for users of your company:
+
+## Request
+
+{% api-method method="get" host="https://api.accp.tradecloud1.com/v2" path="/user-search/_search"%} 
+{% api-method-summary %} Search users {% endapi-method-summary %}
+{% api-method-spec %} 
+{% api-method-request %} 
+{% api-method-headers %} 
+{% api-method-parameter name="Authorization" type="string" required=true %} Authentication token {% endapi-method-parameter %} 
+{% endapi-method-headers %}
+{% api-method-query-parameters %} 
+{% api-method-parameter name="companyId" type="uuid" required=true %} Company Id {% endapi-method-parameter %}
+{% api-method-parameter name="query" type="string" required=false %} Query string {% endapi-method-parameter %}
```

Comment:
Explain the query parameter, which fields can you search on

Reply:
Added description, please verify